test/specs/annotation.tests.ts

```diff
@@ -336,7 +336,7 @@ describe('Annotation counting', () => {
 		annotateAst(ast, context);
 		const [total, annotated] = countQueryBodyAnnotations(ast);
 		console.log(total, annotated);
-		chai.assert.equal(annotated, 3);
+		chai.assert.equal(annotated, 2);
```

Comment:
Magic number 2 what is it?

Reply:
@HectorPeeters  Magic number 2 what is it?

Reply:
Counting annotations. Not really another way than hardcoding them